metadata/datasets/goa.yaml

```diff
@@ -1039,15 +1039,15 @@ datasets:
     - NCBITaxon:9823
 
  -
-   id: goa_pig_rna.gaf
-   label: "goa_pig_rna gaf file"
+   id: goa_pig_rna_noiea.gaf
+   label: "goa_pig_rna_noiea gaf file"
    description: "gaf file for goa_pig_rna from EBI Gene Ontology Annotation Database"
-   url: http://geneontology.org/gene-associations/goa_pig_rna.gaf.gz
+   url: http://geneontology.org/gene-associations/goa_pig_rna_noiea.gaf.gz
    type: gaf
    dataset: goa_pig_rna
```

Comment:
Does this want to be `goa_pig_rna_noiea`?

Reply:
yes